### DISCUSSION TITLE
Nature of ID combinations

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4735] [Ideas]
Often I make identifications through a combination of character diagnoses and geographic distribution, any thoughts on adding the ability to select two natures of ID?

______________________________________________________________

Jonathan L. Dunnum Ph.D.
Senior Collection Manager
Division of Mammals, Museum of Southwestern Biology
University of New Mexico
Albuquerque, NM 87131
(505) 277-9262
Fax (505) 277-1351

MSB Mammals website: http://www.msb.unm.edu/mammals/index.html
Facebook: http://www.facebook.com/MSBDivisionofMammals

Shipping Address:
Museum of Southwestern Biology
Division of Mammals
University of New Mexico
CERIA Bldg 83, Room 204
Albuquerque, NM 87131
________________________________
From: Michelle Koo ***@***.***>
Sent: Monday, November 8, 2021 10:36 PM
To: ArctosDB/arctos ***@***.***>
Cc: Jonathan Dunnum ***@***.***>; Team mention ***@***.***>
Subject: Re: [ArctosDB/arctos] nature_of_id overburdened? (#3940)

  [EXTERNAL]

good topic and clean-up. I'm stealing some of Dusty's response above for some documentation on workflow with respect to CT. Maybe it's a new best practise page? I'll have to think about this, but useful prompt @dustymc<https://github.com/dustymc>
meanwhile I think we can close this

—
You are receiving this because you are on a team that was mentioned.
Reply to this email directly, view it on GitHub<https://github.com/ArctosDB/arctos/issues/3940#issuecomment-963833537>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AED2PA74IHA7DWCZLEKP3XLULCXO7ANCNFSM5EFWAMUA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

_Originally posted by @jldunnum in https://github.com/ArctosDB/arctos/issues/3940#issuecomment-964084016_

**Comment by campmlc** [comment]
I don't see a problem from a collection management point of view. Any
database limitations?

**Comment by dustymc** [comment]
>two natures of ID

That would be Major Surgery.

**Comment by Jegelewicz** [comment]
So add "combos" to the code table?

**Comment by dustymc** [comment]
NO!

**Comment by Jegelewicz** [comment]
OK - so suggestions for handling the situation are appreciated because we obviously have nothing.

**Comment by campmlc** [comment]
No way to select more than one? Allow two nature of IDs?

**Comment by dustymc** [comment]
If two THINGS are necessary then so is different structure. (And "two" is almost never what's really desired- that generally turns out to be either "one" or "many.")

I suspect this is best recorded as two identifications - eg, it looks like {species} and the geography makes it {subspecies} - but I don't have enough information to know if that's the case or not.

Whatever the situation (assuming NoID is actually useful to someone), the "correct" solution isn't overloading/denormalizing a concept.

**Comment by campmlc** [comment]
In this case, the determination is made by both features a geography, both
sources of info are necessary.

**Comment by Jegelewicz** [comment]
> best recorded as two identifications

I get this - but really? EVERYTHING about the two IDs would be exactly the same except for the nature of ID. That seems like a lot of excessive information and it will not translate at all as "I made this ID based upon BOTH features of the object AND geographic information".

**Comment by campmlc** [comment]
So our data structure can't handle the reality of how identifications are
made? That sounds like a problem with the model.
